EKF/ekf.cpp

```diff
@@ -258,6 +243,25 @@ bool Ekf::initialiseFilter()
 	}
 }
 
+bool Ekf::initialiseTilt()
+{
+	if (_accel_lpf.getState().norm() < 0.001f) {
+		return false;
+	}
+
+	// get initial roll and pitch estimate from delta velocity vector, assuming vehicle is static
+	Vector3f gravity_in_body = _accel_lpf.getState();
```

Comment:
```suggestion
	const Vector3f gravity_in_body = _accel_lpf.getState();
```

Reply:
We normalize in the next line. Therefore it can not be constant